Week3/homework/step2-2.js

```diff
@@ -3,18 +3,26 @@
 function threeFive(startIndex, stopIndex, threeCallback, fiveCallback) {
   const numbers = [];
 
-  // Replace this comment and the next line with your code
-  console.log(startIndex, stopIndex, threeCallback, fiveCallback, numbers);
+  for (let i = startIndex; i <= stopIndex; i++) {
+    numbers.push(i);
+  }
+
+  numbers.forEach(number => {
```

Comment:
Since you already use a for loop on line 6 you could move the if statements inside the for loop. The `i` corresponds to the number, you could do `if (i % 3 === 0) {}` inside the for loop. Then you don't need the `forEach` to loop over the array again.